app/client/src/features/events/EventLive.tsx

```diff
@@ -18,11 +18,13 @@ import { SpeakerList } from './Speakers';
 const useStyles = makeStyles((theme) => ({
     root: {
         height: '100%',
+        padding: theme.spacing(0, 3, 0, 3), // add side padding since layout padding is disabled
```

Comment:
This can be rewritten as `theme.spacing(0,3)`.  https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax (theme.spacing has the same syntax rules).